### ISSUE TITLE
Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client

### ISSUE THREAD

**Issue by gromov-io** [isaachinman/next-i18next#815]
## Describe the bug
The error occurs in production mode ' yarn build & & yarn start` 
I use `namespacesRequired` in `getStaticProps`
I received the error 2-3 times, but I can't reproduce it later
When an error occurred, the app continued to work

## Occurs in next-i18next version
```
## System:
 - OS: Linux 5.4 Ubuntu 20.04.1 LTS (Focal Fossa)
 - CPU: (4) x64 Intel(R) Pentium(R) CPU G4600 @ 3.60GHz
 - Memory: 6.65 GB / 15.53 GB
 - Container: Yes
 - Shell: 5.0.17 - /bin/bash
## npmPackages:
 - next: ^9.5.2 => 9.5.2 
 - next-i18next: ^6.0.2 => 6.0.2
```

## Steps to reproduce
`yarn build && yarn start`

## Expected behaviour
working without errors in the console

## Screenshots
![image](https://user-images.githubusercontent.com/46241127/91971480-ed59a400-ed21-11ea-958c-0fe71e904217.png)

**Comment by gromov-io** [comment]
After the localization was added, very long page freezes started in dev mode. Sometimes it can take 20-50 seconds to simply update or navigate through a particular route. SOMETIMES.

**Comment by gromov-io** [comment]
well damn, it is impossible to work, so it is unclear how at the moment to understand why the delay started.

One thing is for sure, it all started after the localization was installed.

It is very disappointing, a lot of time was spent, but instead of a pleasant result, a large number of problems were obtained (

**Comment by isaachinman** [comment]
@gromov-io Can you provide a reproducible repository?

**Comment by balazsorban44** [comment]
Probably related to Next.js. See:
https://github.com/vercel/next.js/issues?q=Error+%5BERR_HTTP_HEADERS_SENT%5D
Has been fixed through https://github.com/vercel/next.js/issues/13868, I cannot remember which version, but updating to the latest Next.js would probably fix that.

**Comment by isaachinman** [comment]
Hi there, I have just released `next-i18next@8.0.0`, which uses the internationalised routing built into NextJs v10+. This new version of `next-i18next` is quite different, thus making this PR no longer relevant. Thanks!
